### DISCUSSION TITLE
Copilot VS2022 Extension . Customizable Font and Color for suggestions

### DISCUSSION THREAD

**Discussion by nanobug** [community/community#106467] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

When using Dark Theme, the in-line code completions come out as almost the exact same color as other code parts.
Sometimes it confuses me whether a semicolon is there or not. It would be nice to let the suggestion stand out more.
VS2022 offers under Tools/Options.. Menu the FONT and COLOR settings. I did not find any category that links to Copilot output there.

**Comment by ayim** [comment]
Hey, I'm going to track this discussion here so that we have an internal ticket for the team as well:
https://developercommunity.visualstudio.com/t/Copilot-VS2022-Extension--Customizable-/10579686

--
Aaron Yim
Product Manager for GitHub Copilot in Visual Studio
github: ayim | aayim@microsoft.com
